client/my-sites/email/inbox-management/mailbox-selection-list/progress-line.jsx

```diff
@@ -0,0 +1,23 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import './progress-line-style.scss';
+
+const ProgressLine = ( { statusText } ) => {
+	return (
+		<div className="progress-line__container">
+			<div className="progress-line__status-text-container">
+				<h2 className="progress-line__status-text"> { statusText } </h2>
```

Comment:
Do we really need those white spaces?

```suggestion
				<h2 className="progress-line__status-text">{ statusText }</h2>
```

Reply:
Removed!